frontend/src/components/CustomerStakesVisualization.tsx

```diff
@@ -1,18 +1,79 @@
 import { FC } from 'react';
 import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
 import Tooltip from '@material-ui/core/Tooltip';
 import { Dot } from './Dot';
 import { percent } from '../utils/string';
 import { CustomerStakes } from '../redux/roadmaps/types';
 import css from './CustomerStakesVisualization.module.scss';
+import colors from '../colors.module.scss';
 
 const classes = classNames.bind(css);
 
+const noRatingsStakes = [
+  { id: 1, value: 0.5 },
+  { id: 2, value: 0.3 },
+  { id: 3, value: 0.1 },
+  { id: 4, value: 0.1 },
+];
+
+const StakeVisualizationTooltip: FC<{
+  title: string;
+  children: any;
```

Comment:
Almost forgot to mention: the `children` is typed by `FC`, so overriding that with less strict type could be dropped
```suggestion
```